core/toolbox.js

```diff
@@ -533,7 +533,9 @@ Blockly.Toolbox.TreeNode = function(toolbox, html, opt_config, opt_domHelper) {
   goog.ui.tree.TreeNode.call(this, html, opt_config, opt_domHelper);
```

Comment:
I'm not a js expert, so I don't really know. A majority of the objects I've written have been subclasses of closure components though.   AFAIU, in this case, doing Blockly.Toolbox.prototype.TreeNode means the TreeNode constructor ends in the Toolbox prototype so to call it you have to say "new Blockly.Toolbox.prototype.TreeNode" which feels awkward.   But this is more a question for @NeilFraser.